From e0b00eb65991bc6bdf196b3e6a005e7c86ac7123 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Rog=C3=A9rio=20Brito?= <rbrito@gmail.com>
Date: Tue, 9 Mar 2021 20:12:30 -0300
Subject: [PATCH 27/27] pdfsizeopt: Show message when we get an image with
 weird bpc.

This should, probably be rewritten as one of the functions that pdfsizeopt
already uses with control for verbosity.
---
 lib/pdfsizeopt/main.py | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/lib/pdfsizeopt/main.py b/lib/pdfsizeopt/main.py
index bddee80..1251418 100644
--- a/lib/pdfsizeopt/main.py
+++ b/lib/pdfsizeopt/main.py
@@ -7343,6 +7343,9 @@ class PdfData(object):
           bpc_has_changed = True
           bpc = 1
       if bpc not in (1, 2, 4, 8):
+
+        print('     **** bpc not in (1, 2, 4, 8): %d' % bpc)
+
         continue
 
       decodeparms = obj.Get('DecodeParms') or ''
-- 
2.30.1

